### PR TITLE
rename table and action names for compatibility with earlier dao.worl…

### PR DESCRIPTION
**Pull request by angelol**
…ds deployment

### CODE REVIEW

File: contracts/daccustodian/README.md [hunk 1]
```diff
@@ -231,9 +231,9 @@ All of the active actions require a `dac_id (account_name)` parameter to be pass
 
 ##### Post Condition:
 
-### nominatecand
+### nominatecane
 
-### nominatecande
+### nominatecanee
```
Comment (dallasjohnson) [inline contracts/daccustodian/README.md:236]: `nominatecanee` should be `nominatecane`

File: contracts/daccustodian/README.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ The candidate should be present in the candidates table and be set to active. If
 
 ---
 
-### withdrawcande
+### withdrawcanee
```
Comment (dallasjohnson) [inline contracts/daccustodian/README.md:262]: `withdrawcanee`should be `withdrawcane`

**Review by dallasjohnson — APPROVED** [review]
2 typos in the README but it's not worth blocking the PR for. 
Thanks for the changes

File: contracts/daccustodian/README.md (hunk 1 above)
Comment (angelol) [reply contracts/daccustodian/README.md]: ✅

File: contracts/daccustodian/README.md (hunk 2 above)
Comment (angelol) [reply contracts/daccustodian/README.md]: ✅

**Pull request merged**